frontend/src/Components/Form.js

```diff
@@ -0,0 +1,161 @@
+import React, { useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { user } from "../Reducers/user";
+import {
+  StyledForm,
+  Label,
+  Input,
+  H1,
+  Wrapper,
+  FormParagraph,
+  HR,
+  P,
+  ErrorText,
+  SecretText,
+} from "styles/Styles";
+
+import { Button } from "./Button";
+
+export const Form = ({ labelHeading, labelText }) => {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+  const accessToken = useSelector(store => store.user.login.accessToken);
+  const userId = useSelector(store => store.user.login.userId);
+  const secretMessage = useSelector(store => store.user.login.secretMessage);
+  const errorMessage = useSelector(store => store.user.login.errorMessage);
+  const dispatch = useDispatch();
+
+  const createUser = event => {
+    event.preventDefault();
+    // const USER_URL = "http://localhost:8080/users";
+    const USER_URL = "https://user-authorisation.herokuapp.com/users";
+
+    fetch(USER_URL, {
+      method: "POST",
+      body: JSON.stringify({ username, password }),
+      headers: { "Content-Type": "application/json" },
+    })
+      .then(res => {
+        if (res.ok) {
+          return res.json();
+        } //eslint-disable-next-line
+        throw "Could not create user"; // we want to display error message from backend here
```

Comment:
To get rid of the eslint error message you have received here can be solved by writing as following: 

`throw new Error(”Could not create user”)`

End in the catch error on line 54 you add 
`dispatch(user.actions.setErrorMessage({ errorMessage: err.toString() }));`